v3/client.go

```diff
@@ -75,15 +75,16 @@ type Client struct {
 	closed    bool
 }
 
+type Client struct{ *internalClient }
```

Comment:
Missing type level comment. 

How does godoc look when you embed `internalClient` like this?

Reply:
Whoops, i'll fix that comment.

Here's what the next PR I'll be submitting (which still just has the wrapped types) looks like after running `godoc` on it:
<img width="1053" alt="Screen Shot 2022-01-19 at 10 46 01 AM" src="https://user-images.githubusercontent.com/87729887/150194608-9567291b-52c0-48c8-a3c4-24b8e2b40b12.png">